vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.h

```diff
@@ -326,11 +328,22 @@ class BatContribution {
       std::unique_ptr<ledger::WalletInfo> info,
       ledger::HasSufficientBalanceToReconcileCallback callback);
 
+  void OnRemovePendingContribution(ledger::Result result);
+
+  void OnContributeUnverifiedPublishers(
+    double balance,
+    const ledger::PendingContributionInfoList& list);
+
+  void OnContributeUnverifiedWallet(
+    ledger::Result result,
+    std::unique_ptr<ledger::WalletInfo> wallet);
+
   bat_ledger::LedgerImpl* ledger_;  // NOT OWNED
   uint32_t last_reconcile_timer_id_;
   uint32_t last_prepare_vote_batch_timer_id_;
   uint32_t last_vote_batch_timer_id_;
   std::map<std::string, uint32_t> retry_timers_;
+  uint32_t unverified_publishers_timer_id_;
```

Comment:
I didn't see where we're initializing this to `0` in the constructor.

Reply:
added